src/state-calculator-form.tsx

```diff
@@ -326,14 +343,33 @@ export const CalculatorForm: FC<{
       .catch(exc =>
         setUtilitiesFetchState({ state: 'error', message: exc.message }),
       );
-  }, [stateId, zip]);
+  };
+
+  useEffect(() => {
+    if (!utilityFetcher) {
+      return;
+    }
```

Comment:
`utilityFetcher` is always defined, no?

Reply:
I believe so, but this check is just a carryover from the existing implementation. https://github.com/rewiringamerica/embed.rewiringamerica.org/pull/231/files#diff-985672295c7571eeac873068c3aa160cfd99ad498905dccb2a74195f7de72b06L277